src/main/java/com/wolox/training/model/Book.java

```diff
@@ -29,12 +32,15 @@
     private Integer pages;
     @NotNull
     private String isbn;
+    @ManyToOne
+    @JoinColumn(name = "user_id")
```

Comment:
use `@ManyToMany`, one user can be associated with many books and one book can be associated with many users